packages/moonstone/TooltipDecorator/Tooltip.module.less

```diff
@@ -50,7 +50,7 @@
 		left: 50%;
 
 		&::after {
-			right: ((@moon-tooltip-point-height / 2) + 1px);
+			right: 81%;
```

Comment:
WDYT about changing the width as suggested by @xbinary?
```
&.centerArrow .tooltipArrow {
    width: ((@moon-tooltip-point-height / 2) + 2px); // If change 2px to 3px, There is no problem on the TV
```

Reply:
That seems like a reasonable adjustment. It's just 1 pixel, and it will make it much more reliable than the existing 2px code. May be good to also change L58 as well, so they both match. Would need to try it out on a TV to verify.

Reply:
I've tried using `+3px` on L58 and there's a visible gap :sadmac:

Reply:
Hah, no worries. just L53 is fine.